content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -241,13 +294,13 @@ Verify the pull command required and run it from a command line. Example: `docke
 
 ![DockerHubCLIInstallTrueCommand](/images/TrueCommand/DockerHubCLIInstallTrueCommand.png "CLI: Install TrueCommand Container")
 
-When the container is downloaded, open the Docker Desktop and select *Images*.
-Hover over the *ixsystems/truecommand* entry to show the **Run** button, then click it.
+When the container downloads, open Docker Desktop and select **Images**.
+Hover over the **ixsystems/truecommand** entry to show the **Run** button, then click it.
 
 ![DockerDesktopImages](/images/TrueCommand/DockerDesktopImages.png "Docker Desktop: Starting the TrueCommand Container")
 
-Open the *Optional Settings* drop down menu, name the container, and set the following port values.
-Investigate your network environment or check with your IT department to ensure that these ports will not conflict with any other running services.
+Open the **Optional Settings** drop-down menu, name the container, and set the following port values.
+Investigate your network environment or check with your IT department to ensure that these ports will not conflict with other running services.
```

Comment:
This can move up to line 302 and bump that line down to 303.
```suggestion
Investigate your network environment or check with your IT department to ensure that these ports do not conflict with other running services.
```